gprofiler/profilers/java.py

```diff
@@ -274,6 +276,37 @@ def get_ap_version() -> str:
 T = TypeVar("T", bound="AsyncProfiledProcess")
 
 
+class ProcessUptimeCalculator:
+    DEFAULT_BINS = [5, 10, 30, 60, 300]
+
+    def __init__(self, process: Process):
+        self._create_time: Optional[float]
+        self._process = process
+        try:
+            self._create_time = process.create_time()
+        except psutil.NoSuchProcess:
+            self._create_time = None
```

Comment:
Why is this value nullable? This makes the code a tad bit more complex. This object is constructed before we start profiling - if the process is dead by then, let the `NoSuchProcess` exception raise (just like we use `process_comm` etc and we don't catch its `NoSuchProcess`)